Update snapshot documentation. (#2133)
firecracker-microvm/firecracker: docs/snapshotting/snapshot-support.md

reason, the wall-clock should be updated to the current time, on the guest-side.
More details on how you could do this can be found at a
[related FAQ](../../FAQ.md#my-guest-wall-clock-is-drifting-how-can-i-fix-it).

### Important notes

Depending on VM memory size, snapshots can consume a lot of disk space. Firecracker 
integrators **must** ensure that the provisioned disk space is sufficient for normal

> operation of their service and also during failure scenarios. If the service exposes

Reviewer: ```suggestion
operation of their service as well as during failure scenarios. If the service exposes
```
